csaf_files/IT/white/2024/va-24-317-01.json

```diff
@@ -0,0 +1,898 @@
+                  "product_id": "CSAFPID-0007"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "22.7R2.1",
+                "product": {
+                  "name": "Ivanti Connect Secure 22.7R2.1",
+                  "product_id": "CSAFPID-0008"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 9.1R18.9",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 9.1R18.9",
+                  "product_id": "CSAFPID-0009"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "9.1R18.9",
+                "product": {
+                  "name": "Ivanti Connect Secure 9.1R18.9",
+                  "product_id": "CSAFPID-0010"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R2.2",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 22.7R2.2",
+                  "product_id": "CSAFPID-0011"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "22.7R2.2",
+                "product": {
+                  "name": "Ivanti Connect Secure 22.7R2.2",
+                  "product_id": "CSAFPID-0012"
+                }
+              }
+            ]
+          },
+          {
+            "category": "product_name",
+            "name": "Policy Secure",
+            "branches": [
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R1",
+                "product": {
+                  "name": "Ivanti Policy Secure 0 < 22.7R1",
+                  "product_id": "CSAFPID-0013"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "22.7R1",
+                "product": {
+                  "name": "Ivanti Policy Secure 22.7R1",
+                  "product_id": "CSAFPID-0014"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R1.1",
+                "product": {
+                  "name": "Ivanti Policy Secure 0 < 22.7R1.1",
+                  "product_id": "CSAFPID-0015"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "22.7R1.1",
+                "product": {
+                  "name": "Ivanti Policy Secure 22.7R1.1",
+                  "product_id": "CSAFPID-0016"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R1.2",
+                "product": {
+                  "name": "Ivanti Policy Secure 0 < 22.7R1.2",
```

Comment:
Change `"name"` field to reflect above change with version range.

Reply:
Fixed in https://github.com/cisagov/CSAF/pull/134/commits/4cc4828ff6a063b14e12da35d64e7c57001938b1